go/chat/convsource.go

```diff
@@ -393,15 +403,18 @@ func (s *HybridConversationSource) GetMessages(ctx context.Context, convID chat1
 	// Form final result
 	var res []chat1.MessageUnboxed
 	for index, msg := range msgs {
+		var m chat1.MessageUnboxed
```

Comment:
sorry, that is an artifact of a previous change.  I'll clean it up to the original.